src/components/Loader/PeopleTable.tsx

```diff
@@ -0,0 +1,71 @@
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from './ PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const mother = people
+            .find(person2 => person.motherName === person2.name);
```

Comment:
person 2 it's not a correct name, it's better to rename it to possibleMother
```suggestion
            .find(possibleMother => person.motherName === possibleMother.name);
```
check other places